src/components/KRadio/KRadio.vue

```diff
@@ -101,34 +126,39 @@ const props = defineProps({
     type: [String, Number, Boolean, Object],
     required: true,
   },
+  hasError: {
+    type: Boolean,
+    default: false,
+  },
+  isCard: {
+    type: Boolean,
+    default: false,
+  },
   /**
-   * Controls appearance of radio input element
+   * @deprecated in favor of `isCard`
    */
   type: {
     type: String as PropType<RadioTypes>,
-    default: 'radio',
-    validator: (value: RadioTypes): boolean => RadioTypesArray.includes(value),
-  },
-  /**
-   * Test mode - for testing only, strips out generated ids
-   */
-  testMode: {
-    type: Boolean,
-    default: false,
+    default: '',
+    validator: (value: RadioTypes): boolean => {
+      if (value) {
+        console.warn('KRadio: `type` prop is deprecated in favor of `isCard`. Please see the migration guide for more details: https://alpha--kongponents.netlify.app/guide/migrating-to-version-9.html#kradio')
+      }
+
+      return RadioTypesArray.includes(value)
+    },
   },
 })
 
 const slots = useSlots()
 
-const inputId = computed((): string => attrs.id ? String(attrs.id) : props.testMode ? 'test-radio-input-id-1234' : uuidv4())
+const inputId = computed((): string => attrs.id ? String(attrs.id) : uuidv4())
 const isDisabled = computed((): boolean => attrs?.disabled !== undefined && String(attrs?.disabled) !== 'false')
 const hasLabel = computed((): boolean => !!(props.label || slots.default))
-const hasDescription = computed((): boolean => !!(props.description || slots.description))
+const showDescription = computed((): boolean => hasLabel.value && (!!props.description || !!slots.description))
+const showCardDescription = computed((): boolean => !!props.label && (!!props.description || !!slots.description))
```

Comment:
nit: The logic for these two computed variables is very similar; can you add comments above each with explaination?

Reply:
Done